### ISSUE TITLE
Don't load documents when dealing with attachments API

### ISSUE THREAD

**Issue by metas-rc** [metasfresh/metasfresh-webui#160]
### Is this a bug or feature request?
* Bug
### What is the current behavior?
* Sometimes, when calling Receive HUs from the left action menu, you get the following error: 

> {timestamp: 1487605745759, status: 500, error: "Internal Server Error",…}
> error
> :
> "Internal Server Error"
> exception
> :
> "de.metas.ui.web.window.exceptions.InvalidDocumentStateException"
> message
> :
> "Document /W540196/1000214 state is invalid: already initializing"
> path
> :
> "/rest/api/window/540196/1000214/attachments"
> status
> :
> 500
> timestamp
> :
> 1487605745759

#### Which are the steps to reproduce?
* Create a purchase order, complete it
* Go to its receipt schedule, to the action menu from the left side (3 points)
* Receive HUs, Create material receipt (repeat several times)
* => NOK: You get the error above at a certain point 
### What is the expected or desired behavior?
* HU Receiving should work with no error

Check the inspection: 
![invalid document state exception](https://cloud.githubusercontent.com/assets/16001442/23132580/be34d2d2-f796-11e6-9901-7a23902b3fa6.png)

**Comment by teosarca** [comment]
stacktrace:

```

2017-02-20 16:16:55.687  WARN 28790 [  84.232.225.151][        it] --- [http-nio-8080-exec-6] d.m.ui.web.config.WebuiExceptionHandler  : Got REST exception from handler=public java.util.List<de.met
as.ui.web.window.datatypes.json.JSONAttachment> de.metas.ui.web.window.controller.WindowRestController.getAttachments(int,java.lang.String)

de.metas.ui.web.window.exceptions.InvalidDocumentStateException: Document /W540196/1000214 state is invalid: already initializing
        at de.metas.ui.web.window.model.Document.initializeFields(Document.java:342)
        at de.metas.ui.web.window.model.Document.refreshFromSupplier(Document.java:714)
        at de.metas.ui.web.window.model.sql.SqlDocumentsRepository.refresh(SqlDocumentsRepository.java:395)
        at de.metas.ui.web.window.model.sql.SqlDocumentsRepository.refresh(SqlDocumentsRepository.java:371)
        at de.metas.ui.web.window.model.Document.refreshFromRepository(Document.java:1565)
        at de.metas.ui.web.window.model.Document.refreshFromRepositoryIfStaled(Document.java:1574)
        at de.metas.ui.web.window.model.DocumentCollection.getRootDocument(DocumentCollection.java:115)
        at de.metas.ui.web.window.model.DocumentCollection.getDocument(DocumentCollection.java:153)
        at de.metas.ui.web.window.controller.WindowRestController.getAttachments(WindowRestController.java:554)
        at sun.reflect.GeneratedMethodAccessor402.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)


```

**Comment by teosarca** [comment]
solution/workaround: avoid loading the Document while fetching the attachments.

**Comment by teosarca** [comment]
Note to IT:
* regression on Swing, Request's attachments
* regression on webui: add, delete, browse

**Comment by metas-rc** [comment]
Results of IT1

* Tested on https://w101.metasfresh.com:8443 and w101-metasfresh ( swing version)

* I stresstested using the steps to reproduce and I didn't get  InvalidDocumentStateException any more. I also tryed from other windows: Orders, Invoice candidates. It went all fine. OK

Regression:
* Tested attachments display, delete, open from both single and grid view. All works fine. OK
* Tested attachments in Request in both swing and webui. All went fine. OK
* Tested attachments in Request by attaching in swing and opening/deleting in webui and vice-versa. All went fine. OK
